variables.tf

```diff
@@ -300,6 +300,12 @@ variable "allow_default_worker_pool_replacement" {
   nullable    = false
 }
 
+variable "enable_ocp_console" {
+  description = "Flag to specify whether to enable or disable the OpenShift console patch."
```

Comment:
Remove the word "patch" from the description

Reply:
done